Dockerfile.mini

```diff
@@ -16,13 +16,17 @@ FROM golang:1.14 as builder
 
 WORKDIR /sigs.k8s.io/external-dns
 
+RUN apt-get update \
+    && apt-get install \
+        ca-certificates \
+    && update-ca-certificates
```

Comment:
what's the need of moving those lines?

Reply:
In docker, you want things that change less often (or can be slow) to happen first. If the packages don't change between builds, why have the cache dump by `COPY . .` affect my package install.